app/test_main.py

```diff
@@ -1,3 +1,55 @@
 from app.main import get_human_age
+# import pytest
 
-# write your code here
+
+def test_get_human_age_for_zeros_in_years() -> None:
+    result = get_human_age(0, 0)
+    expexted_result = [0, 0]
+    assert result == expexted_result
```

Comment:
The assertion uses `expexted_result`, which is a typo. It should be `expected_result`.